cfpb/cfgov-refresh
# Paying for College - School comparison meters

This PR fixes a few things with the Paying for College "Financial Path to Graduation" tool, including the meter charts and some accessibility bugs.

## Changes
- Integrate styles work by virginiacc
- Add handlers for buttons and states for comparison meters
- Improve accessibility/fix accessibility bugs
- Fix direct loans so that they scale with year-over-year caps
- Fix bugs with URL query parsing and calculations of caps before constants model was loaded

## How to test this PR
1. Pull down the PR and get this repo running.
2. Make sure you `./refresh-data.sh`!
3. Browse to http://0.0.0.0:8000/paying-for-college/your-financial-path-to-graduation/ and check for bugs!


## Checklist

### Front-end testing
#### Accessibility

#### Other
